sql-plugin/src/main/330/scala/com/nvidia/spark/rapids/shims/spark330/SparkShimServiceProvider.scala

```diff
@@ -20,9 +20,7 @@ import com.nvidia.spark.rapids.SparkShimVersion
 
 object SparkShimServiceProvider {
   val VERSION = SparkShimVersion(3, 3, 0)
-  // temporarily allow 3.3.1 while 3.3.0 release candidates are being produced
-  private val VERSION331 = SparkShimVersion(3, 3, 1)
-  val VERSIONNAMES = Seq(VERSION, VERSION331).flatMap(v => Seq(s"$v", s"$v-SNAPSHOT"))
+  val VERSIONNAMES = Seq(s"$VERSION", s"$VERSION-SNAPSHOT")
```

Comment:
Snapshot support should have been removed here, posted a fix at #5858.  I don't think it's must-fix for 22.06.

Reply:
thanks!